deployment/CP05MOAS-GL336_Deploy.csv

```diff
@@ -1,55 +1,55 @@
-TS20190516,,UNRECOVERED,,CP05MOAS-GL336-04-DOSTAM000,9,1,2019-05-16T23:37:11,2019-07-28T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-DOSTAM-00542,40.03682,-70.7816,,1000,1000,SS-1;glider lost at sea,,
-TS20190516,,UNRECOVERED,,CP05MOAS-GL336-05-PARADM000,9,1,2019-05-16T23:37:11,2019-07-28T23:59:59,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-PARADM-50163,40.03682,-70.7816,,1000,1000,SS-1;glider lost at sea,,
+CUID_Deploy,deployedBy,CUID_Recover,recoveredBy,Reference Designator,deploymentNumber,versionNumber,startDateTime,stopDateTime,mooring.uid,node.uid,sensor.uid,lat,lon,orbit,deployment_depth,water_depth,notes,electrical.uid,assembly_template_revision
+SI20150110,,MM20150327,,CP05MOAS-GL336-00-ENG000000,1,1,2015-01-11T20:57:00,2015-03-27T21:18:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGCON-GLDRCP-00336,40.02043,-70.38543,,1000,1000,FZ-1,,
+SI20150110,,MM20150327,,CP05MOAS-GL336-01-ADCPAM000,1,1,2015-01-11T20:57:00,2015-03-27T21:18:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-ADCPAM-43106,40.02043,-70.38543,,1000,1000,FZ-1,,
+SI20150110,,MM20150327,,CP05MOAS-GL336-02-FLORTM000,1,1,2015-01-11T20:57:00,2015-03-27T21:18:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-FLORTM-02858,40.02043,-70.38543,,1000,1000,FZ-1,,
+SI20150110,,MM20150327,,CP05MOAS-GL336-03-CTDGVM000,1,1,2015-01-11T20:57:00,2015-03-27T21:18:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-CTDGVM-09030,40.02043,-70.38543,,1000,1000,FZ-1,,
+SI20150110,,MM20150327,,CP05MOAS-GL336-04-DOSTAM000,1,1,2015-01-11T20:57:00,2015-03-27T21:18:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-DOSTAM-00117,40.02043,-70.38543,,1000,1000,FZ-1,,
+SI20150110,,MM20150327,,CP05MOAS-GL336-05-PARADM000,1,1,2015-01-11T20:57:00,2015-03-27T21:18:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-PARADM-50152,40.02043,-70.38543,,1000,1000,FZ-1,,
+CT20160808,,TS20160818,,CP05MOAS-GL336-00-ENG000000,2,1,2016-08-09T13:54:00,2016-08-18T12:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGCON-GLDRCP-00336,39.84253,-70.34048,,1000,1000,FZ-1,,
+CT20160808,,TS20160818,,CP05MOAS-GL336-01-ADCPAM000,2,1,2016-08-09T13:54:00,2016-08-18T12:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-ADCPAM-43106,39.84253,-70.34048,,1000,1000,FZ-1,,
+CT20160808,,TS20160818,,CP05MOAS-GL336-02-FLORTM000,2,1,2016-08-09T13:54:00,2016-08-18T12:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-FLORTM-04194,39.84253,-70.34048,,1000,1000,FZ-1,,
+CT20160808,,TS20160818,,CP05MOAS-GL336-03-CTDGVM000,2,1,2016-08-09T13:54:00,2016-08-18T12:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-CTDGVM-09030,39.84253,-70.34048,,1000,1000,FZ-1,,
+CT20160808,,TS20160818,,CP05MOAS-GL336-04-DOSTAM000,2,1,2016-08-09T13:54:00,2016-08-18T12:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-DOSTAM-00117,39.84253,-70.34048,,1000,1000,FZ-1,,
+CT20160808,,TS20160818,,CP05MOAS-GL336-05-PARADM000,2,1,2016-08-09T13:54:00,2016-08-18T12:45:00,CGVCP-05MOAS-00000,CGVEH-GLDRCP-00336,CGINS-PARADM-50220,39.84253,-70.34048,,1000,1000,FZ-1,,

```

Comment:
Looks correct based on calibration date in Glider 336 Configuration Record, and autoexec record in 3304-00150-00054 FCP.

Reply:
good